### PR TITLE
adds roadmap markdown

### PR DESCRIPTION
**Pull request by stockholmux**
**Issue number:** N/A

Closes # N/A

**Description of changes:**

Adds a markdown document describing the GitHub Project's based roadmap as well as instructions on how it should be used.

**Testing done:** Proofreading

**Terms of contribution:**

By submitting this pull request, I agree that this contribution is dual-licensed under the terms of both the Apache License, version 2.0, and the MIT license.

### CODE REVIEW

File: ROADMAP.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+## How do I follow a specific change to the roadmap?
+
+For individual items, the best way to find out more information is to subscribe to notifications on GitHub for the specific issues linked in the roadmap. This will give you insight into both the development and any large changes to how the issue gets packaged into a release. Additionally, you can ask questions, provide feedback, and, hopefully, contribute. 
+
+## Can I see a log of all the changes to the roadmap?
+
+This was supported in [GitHub Projects classic](https://docs.github.com/en/issues/organizing-your-work-with-project-boards/tracking-work-with-project-boards/tracking-progress-on-your-project-board) however, Bottlerocket uses the newer version of this GitHub feature which lacks this particular log-like view. In the future, it would be great to see this feature return to GitHub Projects or similar functionality through the  [Bottlerocket website](https://github.com/bottlerocket-os/project-website) initiative
```
Comment (stmcginnis) [inline ROADMAP.md]: I'm not sure this last section is really needed.

Also - missing a period at the end.

File: ROADMAP.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Roadmap
+
+The Bottlerocket roadmap is public and hosted as a GitHub Project at the following URL:
+
+https://github.com/orgs/bottlerocket-os/projects/18
+
+GitHub Projects allows for multiple views that represent different ways of looking at the same data. In the case of Bottlerocket, there are four views (represented as tabs):
```
Comment (bcressey) [inline ROADMAP.md]: nit: in other MD docs we've followed a "one line per sentence" convention, which is a little nicer for people using terminal-based editors

**Review by bcressey — APPROVED** [review]

File: ROADMAP.md (hunk 1 above)
Comment (stockholmux) [reply ROADMAP.md]: @stmcginnis This was in reaction to a specific question. 

I'll add that extra period.

File: ROADMAP.md (hunk 2 above)
Comment (stockholmux) [reply ROADMAP.md]: I'll make that change. Have we captured that style preference anywhere?

**Review by stmcginnis — APPROVED** [review]
Great to have some details in the repo to help folks find the project. Looks good!

**Review by zmrow — APPROVED** [review]

File: ROADMAP.md (hunk 2 above)
Comment (bcressey) [reply ROADMAP.md]: Not to my knowledge - is there a logical place for it to go?

**Review by bcressey — APPROVED** [review]

**Review by dmitmasy — APPROVED** [review]

**Pull request merged**